configs/raspberrypi0

```diff
@@ -1,2 +1,9 @@
 BR2_arm1176jzf_s=y
-BR2_LINUX_KERNEL_INTREE_DTS_NAME="bcm2708-rpi-zero"
+BR2_ARM_EABIHF=y
+BR2_LINUX_KERNEL_USE_DEFCONFIG=y
+BR2_LINUX_KERNEL_DEFCONFIG="bcmrpi"
+BR2_LINUX_KERNEL_INTREE_DTS_NAME="bcm2708-rpi-zero-w bcm2708-rpi-zero"
+BR2_PACKAGE_RPI_FIRMWARE_BOOTCODE_BIN=y
+BR2_PACKAGE_RPI_FIRMWARE_VARIANT_PI=y
+BR2_PACKAGE_RPI_FIRMWARE_VARIANT_PI_X=y
+BR2_PACKAGE_RPI_FIRMWARE_CONFIG_FILE="$(BR2_EXTERNAL_PICAM_PATH)/board/config_0.txt"
```

Comment:
Most of these are currently not existing in our setup, can you explain the changes?